README.md

```diff
@@ -57,3 +57,11 @@ WIDTH=4 RADIUS=8 CROSSWALK=True CROSSWALK_LENGTH=2 empy3 templates/intersection.
 ## Resources
 
 Example files with different values are provided under the `resources` folder. 
+
+## Standalone maps
+
+Standalone maps that doesn't respond to a particular template are added in the `standalone` folder.
```

Comment:
"doesn't" should be "don't" or "do not".

"respond" should be "correspond".

Reply:
Oh my bad. Done